best-practices-cel/disallow-empty-ingress-host/disallow-empty-ingress-host.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: disallow-empty-ingress-host
+  annotations:
+    policies.kyverno.io/title: Disallow empty Ingress host
+    policies.kyverno.io/category: Best Practices
+    policies.kyverno.io/minversion: 1.6.0
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Ingress
+    policies.kyverno.io/description: >-
+      An ingress resource needs to define an actual host name
+      in order to be valid. This policy ensures that there is a
+      hostname for each rule defined.
+spec:
+  validationFailureAction: enforce
```

Comment:
This field in the original policy is supposed to be set to Audit right? The Contribution guidelines say this:

> For validate rules, please set validationFailureAction: Audit so that should a user download and apply the policy without having a yet full understanding of Kyverno, it will not cause unintended harm to their environment by blocking resources.

Reply:
I have modified the original policy to include the changes